01-shapefile-attributes.Rmd

```diff
@@ -206,50 +209,35 @@ lines[lines$TYPE == "footpath",]
 
 ```
 
-
-#I think we should plot by attribute HERE - given we are introducing attributes here
-#we should also have some sort of application - maybe we want to find out in a point data set 
-#how many points are of type "X"
-#can we run summary stats on the different categories?
-
-#Plotting Spatial Data in R
-
-We can use the `col` argument to the `plot()` function to specify 
-the color of spatial objects. If type is an attribute of our data, 
-we can use the syntax `$TYPE` to tell `R` to plot the spatial
-objects according to their unique TYPE values. 
+#Plotting Vector Data in R
 
 The following code chunk extracts a vector of this attribute, creates 
 a character color vector of the same length, and changes the positions 
```

Comment:
This is confusing text. Megan i realize this was here - but i'm confused reading it now :)